data/kahet/kahet missions.txt

```diff
@@ -721,3 +721,1128 @@ event "pacified ka'sei"
+			choice
+				`	"Let's go to the storage."`
+					goto storage
+				`	"I think it is best to visit the elevated gardens."`
+					goto gardens
+				`	"Sorry, but the Ka'sei are too much of a danger. I will stay here."`
+			`	"Alright." Dusk and the rest of the team split toward different buildings in the complex, disappearing behind large armored doors. The next hours pass slowly as you stand outside the complex, surrounded by complete silence interrupted only by short updates from those exploring the buildings. Almost four hours later, while checking the atmospheric density scanner yet again, you notice a blinking yellow light, placed near the bottom of the control panel.`
+			choice
+				`	(Call Dusk with the radio.)`
+					goto alarm
+				`	(Ignore it.)`
+			`	As the time passes, the light dims and blinks slower. It takes almost half an hour, but the yellow light finally turns off entirely.`
+			`	Some time later the group returns, satisfied with what they've found inside. "It was an interesting trip for sure, Captain," Dusk tells you. "Did anything of note happen out here?"`
+			choice
+				`	(Tell him about the light.)`
+					goto explain
+				`	"Not much. It was a bit boring."`
+			`	"I imagine so. You would have been more helpful in the complex together with me." After picking up the scanners, you begin to pass through the airlocks, three people at a time. After a few minutes, you're all back outside the dome, ready to finally return to <planet>.`
+				goto ending
+
+			label alarm
+			`	As you tell him about the blinking light, he interrupts you and announces to the rest of the group that they are leaving. "The yellow light is an indicator that the battery is running out," he explains hurriedly. "I should have checked our equipment better. Soon, the starship airlock will be unwatched." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates.`
+				goto ending
+
+			label explain
+			`	As you tell him about the blinking light, his expression changes. "You should have called me! We could have died!" He moves to the console on the laser scanner, which stays dark. "The yellow light was an indicator that the battery was running out," he says. "I should have checked our equipment better. The starship airlock may have remained unwatched for hours now. We need to leave." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates..`
```

Comment:
```suggestion
			`	As you tell him about the blinking light, his expression changes. "You should have called me! We could have died!" He moves to the console on the laser scanner, which stays dark. "The yellow light was an indicator that the battery was running out," he says. "I should have checked our equipment better. The starship airlock may have remained unwatched for hours now. We need to leave." Instead of going through a single airlock, the group splits, half going into one airlock and half into another. When you're finally all outside of the dome, the tense mood dissipates.`
```